### PR TITLE
add subsection 'Configure Eclipse to edit Checker Framework'

### PR DESCRIPTION
**Pull request by h82lin**
Included this subsection under 'Troubleshooting, getting help, and contributing/Build From Source' section in the 'troubleshooting.tex' file to introduce new users in setting up Eclipse to edit the Checker Framework @wmdietl

### CODE REVIEW

File: docs/manual/troubleshooting.tex [hunk 1]
```diff
@@ -974,6 +974,54 @@ \subsection{Build the Checker Framework Manual (this document)\label{building-ma
...
+\item
+Enter the main eclipse working screen and under "File" tab, select Import ->
+General -> Existing Projects into Workspace. Now that the Import Projects window
+appear, tick "Select Root Directory" and browse for these files you have previously
+included in \<\$JSR308>:
+\begin{itemize}
+\item annotation-files-utilities, asmx, checker, dataflow, framework, javacutil,
+jsr308-langtools, scene-lib, stubparser, plume-lib.
+\item Import the above files individually. There might be more files that need to
+be imported depending on which Java file you would like to edit.
```
Comment (Sadaf91) [inline docs/manual/troubleshooting.tex]: remove it

File: docs/manual/troubleshooting.tex [hunk 2]
```diff
@@ -974,6 +974,54 @@ \subsection{Build the Checker Framework Manual (this document)\label{building-ma
 \end{enumerate}
 
 
+\subsection{Configure Eclipse to edit the Checker Framework\label{building-eclipse}}
+
```
Comment (Sadaf91) [inline docs/manual/troubleshooting.tex]: remove

File: docs/manual/troubleshooting.tex [hunk 3]
```diff
@@ -974,6 +974,54 @@ \subsection{Build the Checker Framework Manual (this document)\label{building-ma
...
+\item At this point the tools.jar file isn't
+in the plume-lib directory. A way to resolve this issue is by creating a
+softlink of \<JAVA\_HOME> in the plume-lib directory, due to the fact that tools.jar
+is just a lib jar in the \<JAVA\_HOME> directory. Here is the command to create
+the softlink:
+\begin{Verbatim}
+cd $JSR308/plume-lib
+ln -s /usr/lib/jvm/java-8-openjdk-amd64 jdk
+\end{Verbatim}
+
```
Comment (Sadaf91) [inline docs/manual/troubleshooting.tex]: remove

File: docs/manual/troubleshooting.tex [hunk 4]
```diff
@@ -974,6 +974,54 @@ \subsection{Build the Checker Framework Manual (this document)\label{building-ma
...
+included in \<\$JSR308>:
+\begin{itemize}
+\item annotation-files-utilities, asmx, checker, dataflow, framework, javacutil,
+jsr308-langtools, scene-lib, stubparser, plume-lib.
+\item Import the above files individually. There might be more files that need to
+be imported depending on which Java file you would like to edit.
+\end{itemize}
+
+\item
+After all projects have been imported, red error messages should not appear. Please
```
Comment (Sadaf91) [inline docs/manual/troubleshooting.tex]: everything should build successfully.

Comment (h82lin) [reply docs/manual/troubleshooting.tex]: You mean to close this pull request?

Comment (wmdietl) [reply docs/manual/troubleshooting.tex]: @Sadaf91 is taking a pass through this PR and will push some updates. Once we're all happy with the changes, we'll close this PR and move it to typetools. Here comments here were reminders what she needs to change.

Comment (h82lin) [reply docs/manual/troubleshooting.tex]: Okay sounds good!

**Comment by Sadaf91** [comment]
Closing this PR, as it is replaced by PR #59

**Pull request closed without merging**